java/src/main/java/ai/pluggy/quickstart/QuickstartApplication.java

```diff
@@ -0,0 +1,122 @@
+package ai.pluggy.quickstart;
+
+import java.io.IOException;
+import java.text.MessageFormat;
+import java.util.List;
+
+import ai.pluggy.client.PluggyClient;
+import ai.pluggy.client.request.CreateItemRequest;
+import ai.pluggy.client.request.ParametersMap;
+import ai.pluggy.client.response.Account;
+import ai.pluggy.client.response.Connector;
+import ai.pluggy.client.response.IdentityResponse;
+import ai.pluggy.client.response.ItemResponse;
+import ai.pluggy.client.response.Transaction;
+import retrofit2.Response;
+
+public class QuickstartApplication {
+
+  public static void main(String[] args) throws IOException, InterruptedException {
+    System.out.println("Hello World");
+
+    // Client configurations
+    final String clientId = "5d4f614a-0809-4341-a986-1b7bbdaf0668";
+    final String clientSecret = "a7aa3281-0b89-42e7-8b22-efdac71d5cbd";
+    final String webhookUrl = "https://pluggy-java.requestcatcher.com";
+
+    // Pluggy bank connector
+    final int pluggyBankConnectorId = 2;
```

Comment:
could we use a constant here to document what this `2` value means?

Reply:
How's that?